.env.local.sample

```diff
@@ -0,0 +1,6 @@
+AUTH0_SECRET='use [openssl rand -hex 32] to generate a 32 bytes value'
+AUTH0_BASE_URL='http://localhost:3000'
+AUTH0_ISSUER_BASE_URL='https://YOUR_DOMAIN'
+AUTH0_CLIENT_ID='YOUR_CLIENT_ID'
+AUTH0_CLIENT_SECRET='YOUR_CLIENT_SECRET'
```

Comment:
are these lines needed as part of this issue? I think they should already be there from Auth0 integration PR

Reply:
yes. I see that this file was removed from master . We need to have this to keep track of all config variables we eventually use in our app. In Auth0 I had not added the readme hence added it here.